cockpit/package.json

```diff
@@ -67,5 +67,8 @@
     "hoek": "^4.2.1",
     "moment": "^2.19.3",
     "lodash": "^4.17.13"
+  },
+  "engines": {
+    "npm": "*"
```

Comment:
probably should lock this down to 5.10.0 or newer (i used 6.14.8)

Reply:
I think that is a good idea, something we could even change in each of the support branches.